commit 1340b957ffb21ebd48454efa1f702151dcfa2099
Author: Guenter Roeck <linux@roeck-us.net>
Date:   Wed Feb 8 14:07:42 2017 -0800

    hwmon: (it87) Do not overwrite bit 2..6 of pwm control registers
    
    commit 4c7b8ca1ae5ed9e27014732c8a918ba11a86cf09 upstream.
    
    In IT8620E, after setting pwm control to manual, it was observed that
    pwm values for fan 4..6 have reversed results (writing 0 results in fans
    running at full speed, writing 255 results in fans turned off).
    
    With the new PWM control, pwm polarity for pwm control 4..6 is specified
    in its pwm control registers. Those registers are overwritten when setting
    the pwm mode or the temperature mapping. Do not touch bit 2..6 of pwm
    control registers on register writes to fix the problem.
    
    Signed-off-by: Guenter Roeck <linux@roeck-us.net>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1239_1340b957ffb21ebd48454efa1f702151dcfa2099.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1239_1340b957ffb21ebd48454efa1f702151dcfa2099.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
